hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Invoker.java

```diff
@@ -471,7 +471,8 @@ public <T> T retryUntranslated(
       }
       // you only get here if the operation didn't complete
       // normally, hence caught != null
-
+      LOG.debug("{} ; {}, ", text, caught.toString());
+      LOG.trace("{}", caught);
```

Comment:
doubt: Is there some string we intended to add for tracing before the Exception stack trace? Else we can remove the placeholder "{}".

Reply:
Yes you are right.